docs/test_tutorials.py

```diff
@@ -21,6 +21,7 @@
 _exclude_files = ['tutorial_external_solver.rst']
```

Comment:
Should this `rst` also be changed to `md`?

Reply:
Good point. Basically that I forget that change means that the exclusion is no longer necessary though.